tests/tensor_search/integ_tests/test_hybrid_search.py

```diff
@@ -1274,49 +1620,77 @@ def test_hybrid_search_none_query_with_context_vectors_passes(self):
+            tensor_search.search(
+                config=self.config,
+                index_name=self.unstructured_default_text_index.name,
+                text="dogs",
+                search_method="HYBRID",
+                hybrid_parameters=HybridParameters(
+                    retrievalMethod=RetrievalMethod.Tensor,
+                    rankingMethod=RankingMethod.Tensor,
+                    searchableAttributesTensor=["text_field_1"]
+                )
+            )
+        self.assertIn("does not support `searchableAttributesTensor` or `searchableAttributesLexical`",
+                      str(e.exception))
+
+    def test_hybrid_search_unstructured_with_2_10_fails(self):
         """
-        Test that hybrid parameters with wrong search method fails.
+        Test that hybrid search with unstructured index with version 2.10.0 fails (version is below the minimum).
         """
 
-        # TODO: Use api.search() instead of tensor_search.search()
-        # Covered in API tests
-        pass
+        with self.assertRaises(core_exceptions.UnsupportedFeatureError) as e:
+            tensor_search.search(
+                config=self.config,
+                index_name=self.unstructured_index_2_10.name,
+                text="dogs",
+                search_method="HYBRID",
+            )
+        self.assertIn("only supported for Marqo unstructured indexes created with Marqo 2.11.0 or later",
+                      str(e.exception))
 
-    # TODO: Remove when unstructured index is supported
-    def test_hybrid_search_on_unstructured_index_fails(self):
+    def test_hybrid_search_structured_with_2_9_fails(self):
         """
-        Test that hybrid search on an unstructured index fails.
+        Test that hybrid search with structured index with version 2.9.0 fails (version is below the minimum).
         """
 
         with self.assertRaises(core_exceptions.UnsupportedFeatureError) as e:
             tensor_search.search(
                 config=self.config,
-                index_name=self.unstructured_default_text_index.name,
+                index_name=self.structured_index_2_9.name,
                 text="dogs",
                 search_method="HYBRID",
-                hybrid_parameters=HybridParameters(
-                    retrievalMethod=RetrievalMethod.Disjunction,
-                    rankingMethod=RankingMethod.RRF,
-                    alpha=0.6
-                )
             )
-        self.assertIn("not yet supported for hybrid search", str(e.exception))
+        self.assertIn("only supported for Marqo structured indexes created with Marqo 2.10.0 or later",
+                      str(e.exception))
 
+    def test_hybrid_parameters_with_wrong_search_method_fails(self):
+        """
+        Test that hybrid parameters with wrong search method fails.
+        """
+
+        # TODO: Use api.search() instead of tensor_search.search()
+        # Covered in API tests
+        pass
```

Comment:
nit. This file is super large now. For better maintenance, consider extracting common methods or break it down to multiple files. Does not need to be done in this PR.

Reply:
Noted, this is probably best done by extracting common logic, maybe in a future refactor.